base/array.jl

```diff
@@ -713,25 +748,27 @@ Resize `a` to contain `n` elements. If `n` is smaller than the current collectio
 length, the first `n` elements will be retained. If `n` is larger, the new elements are not
 guaranteed to be initialized.
 
+# Examples
 ```jldoctest
 julia> resize!([6, 5, 4, 3, 2, 1], 3)
 3-element Array{Int64,1}:
  6
  5
  4
-```
 
-```julia-repl
-julia> resize!([6, 5, 4, 3, 2, 1], 8)
-8-element Array{Int64,1}:
- 6
- 5
- 4
- 3
- 2
- 1
- 0
- 0
+ julia> a = resize!([6, 5, 4, 3, 2, 1], 8);
```

Comment:
how strange, I see bdbcbba from a fetch, but github isn't showing it in the pr yet?